cmd/helm/init.go

```diff
@@ -118,6 +119,7 @@ func newInitCmd(out io.Writer) *cobra.Command {
 	f.BoolVarP(&i.clientOnly, "client-only", "c", false, "If set does not install Tiller")
 	f.BoolVar(&i.dryRun, "dry-run", false, "Do not install local or remote")
 	f.BoolVar(&i.skipRefresh, "skip-refresh", false, "Do not refresh (download) the local repository cache")
+	f.BoolVar(&i.skipRepos, "skip-repo", false, "Skip adding the stable and local repositories")
```

Comment:
should this be `--skip-repos` to align with the variable name `skipRepos`? Given this skips adding multiple repositories, I think the plurality makes sense here

Reply:
oops. fixed. good catch.